pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -1487,6 +1498,33 @@ static void updateColumnNames(String rawTableName, PinotQuery pinotQuery, boolea
     }
   }
 
+  private static void expandStarExpressionsToActualColumns(PinotQuery pinotQuery, Map<String, String> columnNameMap,
+      Expression selectStarExpr) {
```

Comment:
Same here, no need to pass in this expression. Use constant instead

Reply:
Addressed.